### PR TITLE
Feature/keycard   create pin #21368

### LINKED ISSUE #21368: Keycard - Create pin
image

Figma: https://www.figma.com/design/YGm3igIOAcwMqUVJWCJ6f1/Keycard-UX?node-id=386-49535&t=WC8Kdtt0D1tEpdGs-1

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #21368

QA : because there is no changes in the current code, this PR will be merged without e2e

### CODE REVIEW

File: src/status_im/contexts/keycard/pin/create/view.cljs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                                               (set-pin "")
+                                               (set-first-pin new-pin)
+                                               (set-stage :repeat)))))))
+                                   [pin stage first-pin])]
+    [rn/view {:style {:padding-bottom 12 :flex 1}}
+     [quo/drawer-top
+      {:title (if (= :create stage)
+                (i18n/label :t/create-keycard-pin)
+                (i18n/label :t/repeat-keycard-pin))}]
+     [rn/view {:style {:flex 1 :justify-content :center :align-items :center :padding-vertical 34}}
```
Comment (Parveshdhull) [inline src/status_im/contexts/keycard/pin/create/view.cljs:45]: Hi @flexsurfer, Thank you for your PR.

Please format style.

File: src/status_im/contexts/keycard/pin/create/view.cljs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+(defn view
+  [{:keys [on-complete]}]
+  (let [[pin set-pin]             (rn/use-state "")
+        [first-pin set-first-pin] (rn/use-state "")
+        [error? set-error]        (rn/use-state false)
+        [stage set-stage]         (rn/use-state :create)
+        pin-empty?                (string/blank? pin)
+        on-delete                 (rn/use-callback
+                                   (fn []
```
Comment (Parveshdhull) [inline src/status_im/contexts/keycard/pin/create/view.cljs:16]: Maybe we should create functions separately for improving readability. wdyt?

```
on-delete (rn/use-callback #(on-delete-fn pin) [pin])
```

**Review by Parveshdhull — APPROVED** [review]

File: src/status_im/contexts/keycard/pin/create/view.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/contexts/keycard/pin/create/view.cljs:45]: wdym? lint is ok

File: src/status_im/contexts/keycard/pin/create/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/contexts/keycard/pin/create/view.cljs:16]: thank you, yeah i thought about that , do you think it's better?

Comment (flexsurfer) [reply src/status_im/contexts/keycard/pin/create/view.cljs:16]: i can do it in the next PRs

File: src/status_im/contexts/keycard/pin/create/view.cljs (hunk 1 above)
Comment (Parveshdhull) [reply src/status_im/contexts/keycard/pin/create/view.cljs:45]: I think the style will be more readable if we have one key value per line (especially if the number key is more than two). Currently, it seems a little harder to differentiate which is key and which is value.

**Pull request merged**